### ISSUE TITLE
feature: Show video duration to the thumbnail instead of 0

### ISSUE THREAD

**Issue by mvp5464** [code100x/cms#485]
**Is your feature request related to a problem? Please describe.**
In the thumbnail of the video it shows 0 instead of real time duration.

**Describe the solution you'd like**
It should show real duration of the video

**Describe alternatives you've considered**
Seed the database with true duration of the video

**Additional context**
![Screenshot](https://github.com/code100x/cms/assets/28997328/cadec24d-1fcd-4acf-85cf-cd7c6d5a37ac)

**Comment by mvp5464** [comment]
@hkirat sir, tried many methods (like seeding all videos in SSG, etc.) to seed duration, but this seems good without any extra db call. After old video duration is seeded then we can add duration when you upload the video (working on it).

- any suggestion or improvement sir.

**Comment by aryan-17** [comment]
So do you want to remove that zero on thumbnails??

**Comment by mvp5464** [comment]
No, It is supposed to have total video duration on it (same as youtube) but the database is empty so to fill the db and then fetch that data here.

**Comment by manthan-sharma-23** [comment]
I guess it is showing in some places and not in some places

**Comment by hkirat** [comment]
yeah we have a json file with the durations just need to seed the DB
I think someone created the code to seed it

**Comment by aryan-17** [comment]
Is this issue closed then?

**Comment by mvp5464** [comment]
@hkirat sir, 
- I think you are saying about this PR #392, it seeds the DB by running a command, 
- PR #486, does it without any command or any json file.  (I thought we don't have a json file with duration so created this)

**Comment by mvp5464** [comment]
> Is this issue closed then?
> 
> I think the pr raised #486 has resolved this issue but still needs to clear the linting tests. That's the thing still pending

@aryan-17 Ya created by me, but json file is already there so may be not needed this pr
